frontend/coprs_frontend/coprs/views/coprs_ns/coprs_general.py

```diff
@@ -876,8 +876,17 @@ def _render_external_repo_template(dep, copr_dir, mock_chroot, dep_idx):
                                  url=baseurl) + "\n"
 
 
-@cache.memoize(timeout=5*60)
 def render_generate_repo_file(copr_dir, name_release, arch=None):
+    name = helpers.get_stat_name(
+        CounterStatType.REPO_DL,
+        copr_dir=copr_dir,
+        name_release=name_release,
+    )
+    CounterStatLogic.incr(name=name, counter_type=CounterStatType.REPO_DL)
+    return render_generate_repo_file_cached(copr_dir, name_release, arch=arch)
```

Comment:
The `commit()` call should be moved as well.

Reply:
If you merge https://github.com/fedora-copr/copr/pull/3074 then incr() will do the commit and it is not necessary to add it here.

Reply:
Updated